src/main/java/org/sonarsource/sonarlint/ls/DiagnosticPublisher.java

```diff
@@ -166,6 +155,17 @@ public static void setSource(Issue issue, Diagnostic diagnostic) {
     }
   }
 
+  private static void setData(Diagnostic diagnostic, Issue issue, String entryKey) {
+    var data = new DiagnosticData();
+    if (issue instanceof DelegatingIssue) {
+      var delegatedIssue = (DelegatingIssue) issue;
+      data.setStatus(delegatedIssue.getReviewStatus());
```

Comment:
Just to double-check, the `getReviewStatus` will give valid results for issues AND hotspots right?